frontend/public/components/utils/managed-by.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { ResourceIcon } from './resource-icon';
+import {
+  K8sResourceCommon,
+  K8sResourceKind,
+  referenceForGroupVersionKind,
+  referenceForModel,
+  referenceForOwnerRef,
+  OwnerReference,
+} from '../../module/k8s';
+import { RootState } from '@console/internal/redux';
+import { withFallback } from '@console/shared/src/components/error/error-boundary';
+import { ClusterServiceVersionModel } from '../../models';
+
+const managedByLinkStateToProps = (state: RootState): managedByLinkStateProps => {
+  return {
+    activeNamespace: state.UI.get('activeNamespace'),
+    receivedCSVs: state.k8s.getIn(['CSVS', 'csvs']),
+  };
+};
+
+const isOwnedByOperator = (csv: K8sResourceKind, owner: OwnerReference) => {
+  return (
+    csv.spec.customresourcedefinitions.owned.find((api) => api.kind === owner.kind) ===
+    csv.spec.customresourcedefinitions.owned.find((api) => owner.apiVersion.includes(api.version))
```

Comment:
Don't use `includes`. We need to make sure the expected API version matches exactly.